geode-benchmarks/build.gradle

```diff
@@ -23,16 +23,29 @@ version '1.0-SNAPSHOT'
 sourceCompatibility = 1.8
 
 def outputDir = project.hasProperty('outputDir') ? project.findProperty('outputDir') : new File(project.buildDir, "benchmarks_" + getDate()).getAbsolutePath()
-def geodeVersion = project.hasProperty('geodeVersion') ? project.findProperty('geodeVersion') : '1.+'
+def geodeVersion = project.hasProperty('geodeVersion') ? project.findProperty('geodeVersion') : '1.13.+'
```

Comment:
this repo now depends on Geode 1.13-or-later because that version added SNI support

Reply:
This will actually pin to 1.13.x not 1.13+. Just leave this change out. There are some combinations that don't work with some versions of Geode anyway.

Reply:
I hear two suggestions:

1. that the version range spec is wrong for its intended purpose, that maybe it should be `1.13+`
2. that the version range spec should be widened (to what it was before) to include older versions of Geode

On (2) since `geodeVersion` is used to specify a compile-time dependency, if it is reverted to `1.+` then it'll allow versions of Geode that will prevent compilation of this module. The whole point here is to specify the minimum required Geode version. And that version is `1.13.0` right?

If we all buy that then on (1): is the suggestion to use `1.13+` instead of `1.13.+`?

Reply:
The latest commit updates the range spec to `[1.13,)`

Reply:
No, drop the version restriction completely.

Reply:
> since geodeVersion is used to specify a compile-time dependency, if it is reverted to 1.+ then it'll allow versions of Geode that will prevent compilation of this module. The whole point here is to specify the minimum required Geode version. And that version is 1.13.0 right?

Reply:
This change prevents this benchmark from baselining older versions of Geode. Let's look at using reflection for now. Perhaps a plug-in model for future.